src/content.js

```diff
@@ -1,43 +1,78 @@
 (() => {
-    let unreadCount;
-
-    function getUnreadCount(doc) {
-        if (!doc) return -1;
-        const fullcountElement = doc.querySelector('fullcount');
-        if (!fullcountElement) return -1;
-        const count = parseInt(fullcountElement.textContent);
-        return isNaN(count) ? -1 : count;
-    }
+  let unreadCount;
+
+  function getUnreadCount(doc) {
```

Comment:
I have submitted an other PR where I am changing this function to be based on `entry` due to an issue with full count and categories/pre defined labels.

Have a look [here](https://github.com/aberonni/gmail-app-badge-notification/pull/27/files#diff-1f93cd4783614fe92bbb9d7fba4cd2b31ba1a47c0c2e9b00be36bd3246e5af7fR6-R7) in case we can combine the PRs.

Having said that, I am not the owner of the repo/extension, so not sure if that's acceptable overall in regards to changing the function